### PR TITLE
[CTOR-610] Refresh Veeam doc page

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-nrpe.md [hunk 1]
```diff
@@ -5,87 +5,510 @@ title: Veeam NSClient++ NRPE
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-nrpe.md:7]:
```suggestion

The *Veeam* Monitoring Connector works with the Centreon NSClient++ monitoring agent and 
Powershell to check the operating status of a Veeam Server.

```

File: pp/integrations/plugin-packs/procedures/applications-veeam-nrpe.md [hunk 2]
```diff
@@ -5,87 +5,508 @@ title: Veeam NSClient++ NRPE
...
 
-| Mandatory | Name             | Description                                                      |
-|:----------|:-----------------|:---------------------------------------------------------------- |
-| X         | NRPECLIENT       | NRPE Plugin binary to use (Default: 'check_centreon_nrpe')       |
-| X         | NRPEPORT         | NRPE Port of the target server (Default: '5666')                 |
-| X         | NRPETIMEOUT      | Timeout value (Default: '30')                                    |
-| X         | NRPEEXTRAOPTIONS | Extraoptions to use with the NRPE binary (default: '-u -m 8192') |
\ No newline at end of file
+```bash
+/usr/lib64/nagios/plugins//check_centreon_nrpe -H 10.0.0.1 -p 5666 -t 55  -c check_centreon_plugins -a 'apps::backup::veeam::local::plugin' 'tape-jobs'  ' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-veeam-nrpe.md:341]: problem here too

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-nrpe.md [hunk 3]
```diff
@@ -5,87 +5,510 @@ title: Veeam NSClient++ NRPE
...
-|:----------|:-----------------|:---------------------------------------------------------------- |
-| X         | NRPECLIENT       | NRPE Plugin binary to use (Default: 'check_centreon_nrpe')       |
-| X         | NRPEPORT         | NRPE Port of the target server (Default: '5666')                 |
-| X         | NRPETIMEOUT      | Timeout value (Default: '30')                                    |
-| X         | NRPEEXTRAOPTIONS | Extraoptions to use with the NRPE binary (default: '-u -m 8192') |
\ No newline at end of file
+```bash
+/usr/lib64/nagios/plugins//check_centreon_nrpe -H 10.0.0.1 -p 5666 -t 55  -c check_centreon_plugins -a 'apps::backup::veeam::local::plugin' 'tape-jobs'  ' \
+	--filter-name="" \
+	--list-mode
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-nrpe.md]:
```suggestion
	--list-mode'
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-nrpe.md (hunk 1 above)
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-veeam-nrpe.md:7]:
```suggestion

Le connecteur de supervision *Veeam* fonctionne avec l'agent de supervision Centreon NSClient++ et Powershell, et contrôle le statut opérationnel d'un serveur Veeam.

```

File: pp/integrations/plugin-packs/procedures/applications-veeam-nrpe.md [hunk 4]
```diff
@@ -5,87 +5,511 @@ title: Veeam NSClient++ NRPE
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+The *Veeam* Monitoring Connector works with the Centreon NSClient++ monitoring agent and 
+Powershell to check the operating status of a Veeam Server.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-veeam-nrpe.md:9]:
```suggestion
Powershell to check the operating status of a Veeam server.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**